src/components/CircleProgressBar/index.js

```diff
@@ -17,19 +17,32 @@ const CircleProgressBar = React.createClass({
 
   componentDidMount () {
     this.initProgressBar();
-    this.updateProgressBar(this.props.percentage);
+    if (this.props.percentage > 0) {
+      this.updateProgressBar(this.props.percentage);
+    }
```

Comment:
ping @artibella - only init the bar if `percentage > 0` (stop weird flash of the line then disappearing when equals 0)